API/src/main/java/xyz/kyngs/librelogin/api/server/ServerHandler.java

```diff
@@ -11,6 +11,7 @@
 import xyz.kyngs.librelogin.api.database.User;
 
 import java.util.Collection;
+import java.util.Optional;
```

Comment:
Unused import?

Reply:
Fixed this in [9e7d402](https://github.com/kyngs/LibreLogin/pull/103/commits/9e7d4023429dcf904d03cd0887faf67d7ffdbfb2)